### PR TITLE
Fix python3 incompatibilities

### PR DESCRIPTION
**Pull request by ron8mcr**
Update `tox.ini` and fix incompabilities with Python3

### CODE REVIEW

**Comment by bashu** [comment]
@praekelt sup?

**Comment by bashu** [comment]
@hedleyroos, @qoda can you take a look at this? thanks!

**Comment by bashu** [comment]
...and if you are all busy, why not add me as collaborator and I will  help prepare next release

File: object_tools/tests/__init__.py [hunk 1]
```diff
@@ -49,41 +51,44 @@ def setUpClass(cls):
...
+        self.assertRaises(
             NotImplementedError, validate, TestInvalidTool, self.user_klass
         )
         try:
             validate(TestInvalidTool, self.user_klass)
-        except NotImplementedError, e:
-            self.failUnlessEqual(
-                e.message, "No 'view' method found for tool TestInvalidTool."
+        except NotImplementedError as e:
+            message = e.args[0]
```
Comment (phala) [inline object_tools/tests/__init__.py]: Why aren't we using `str(e)`?

File: object_tools/tests/__init__.py [hunk 2]
```diff
@@ -49,41 +51,44 @@ def setUpClass(cls):
...
+        self.assertRaises(
             ImproperlyConfigured, validate, TestInvalidTool, self.user_klass
         )
         try:
             validate(TestInvalidTool, self.user_klass)
-        except ImproperlyConfigured, e:
-            self.failUnlessEqual(
-                e.message,
+        except ImproperlyConfigured as e:
+            message = e.args[0]
```
Comment (phala) [inline object_tools/tests/__init__.py]: Why aren't we using `str(e)`?

File: object_tools/tests/__init__.py [hunk 3]
```diff
@@ -49,41 +51,44 @@ def setUpClass(cls):
...
+        self.assertRaises(
             ImproperlyConfigured, validate, TestInvalidTool, self.user_klass
         )
         try:
             validate(TestInvalidTool, self.user_klass)
-        except ImproperlyConfigured, e:
-            self.failUnlessEqual(
-                e.message, "No 'name' attribute found for tool TestInvalidTool."
+        except ImproperlyConfigured as e:
+            message = e.args[0]
```
Comment (phala) [inline object_tools/tests/__init__.py]: Why aren't we using `str(e)`?

**Comment by phala** [comment]
Hi @ron8mcr and @bashu.
I've left some feedback for the proposed changes :)

**Comment by phala** [comment]
@ron8mcr merged. Thanks

**Pull request merged**